client/tailscale/localclient.go

```diff
@@ -1124,6 +1124,25 @@ func (lc *LocalClient) DeleteProfile(ctx context.Context, profile ipn.ProfileID)
 	return err
 }
 
+// QueryFeature makes a request for instructions on how to enable a
+// feature, such as Funnel, for the node's tailnet.
+//
+// This request itself DOES NOT directly enable the feature on behalf
```

Comment:
I think this shouting is probably no longer necessary given the new name of the method.